src/bundle/Resources/views/themes/admin/account/change_password/form_fields.html.twig

```diff
@@ -1,17 +1,7 @@
 {# @todo move to view/user/preferences in the future #}
 
-{% use 'bootstrap_4_layout.html.twig' %}
```

Comment:
Review note: dropping this allows proper inheritance of form themes in `src/bundle/Resources/views/themes/admin/account/change_password/index.html.twig`